projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/shipping-address/address-form/address-form.component.spec.ts

```diff
@@ -181,7 +182,7 @@ describe('AddressFormComponent', () => {
       .unsubscribe();
 
     expect(countries).toBe(mockCountries);
-    expect(titles).toBe(mockTitles);
+    expect(titles).toEqual([{ code: '', name: 'None' }, ...mockTitles]);
```

Comment:
you should rather add this 'none' titleCode to the mockTitles array, am I right?

Reply:
not really, then 'none' option will be duplicated. But I can separate it to another array